CHANGELOG.md

```diff
@@ -19,11 +19,15 @@
 - Potentially breaking: `RArray::from_parts` no longer requires a pointer to the underlying data vector [[#657]](https://github.com/extendr/extendr/pull/657)
 - Breaking change: In `AltRep` the `unserialize_ex`, `set_parent`, `set_envflags` are
 now hidden behind the feature flag `non-api`. Also, `Promise::from_parts` is marked as non-API.
+- Floating point numbers with decimal part can no longer be converted to integer types via
```

Comment:
I'd call this a breaking change.

Reply:
It certainly is. What do you want me to do?

Reply:
Nothing! We starting prefacing these things with breaking change 🤷‍♂️

Reply:
What would you want me to do?

Reply:
Write: Breaking Change:

Reply:
Sorry, sent one message from the phone and another from my PC. Let me fix it